Source/Core/VideoBackends/OGL/VertexManager.cpp

```diff
@@ -140,6 +143,13 @@ void VertexManager::vFlush(bool useDstAlpha)
 
 	PrepareDrawBuffers(stride);
 
+	if (!bpmem.genMode.zfreeze)
+		CalculateZSlope(stride);
+
+	// If cull mode is CULL_ALL, do not render these triangles
```

Comment:
Improve comment to talk about quads.

Can we move this check back into videocommon?